### DISCUSSION TITLE
Can't find OPC_UA server from Examples with Matricon OPC Explorer or SCADA OPC client

### DISCUSSION THREAD

**Discussion by VasilyBroze** [FreeOpcUa/opcua-asyncio#1134] [Q&A]
Hello, I tried to connect clients to the server, but the server does not appear in the list of available computers in clients (other OPC servers are visible). Is it possible? May be it's some settings needed to be visible in list?
Incriment is working on server, and i was sucsessfully connected with my phone (without explorer, just by ip).
Thank you!

**Comment by schroeder-** [comment]
The server must server on Port 4840, this is the port where clients are looking for.
